fineract-provider/src/main/java/org/apache/fineract/infrastructure/jobs/filter/LoanCOBFilterHelper.java

```diff
@@ -217,7 +227,7 @@ private List<Long> getLoanIdsFromBatchApi(BodyCachingHttpServletRequestWrapper r
             // check the body for Loan ID
             Long loanId = getTopLevelLoanIdFromBatchRequest(batchRequest);
             if (loanId != null) {
-                if (isLoanHardLocked(loanId)) {
+                if (isLoanHardLocked(loanId) && isLockOverrulable(loanId)) {
```

Comment:
Wrong condition. If the lock has an error stored, we should allow to run the inline cob!